extensions/activemq/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1,2 @@
+ActiveMQ https://activemq.apache.org/selectors.html[XPath selectors] are not supported in native mode as the libraries to support it are not on the classpath
+by default.
```

Comment:
Hm... this sounds like they could be made to work if one adds the necessary libs. But it is obviously not true due to the substitution. Wouldn't something like "We have explicitly disabled ActiveMQ https://activemq.apache.org/selectors.html[XPath selectors] in the native mode because they would not work anyway, because of [the reason]" be more honest from us?

Reply:
Well, it would require adding `activemq-broker` to the classpath. Which is not something we want to support, and wont work natively by default (which is why we exclude it from the dep tree).

I can add a comment to that affect.